build.gradle

```diff
@@ -34,6 +34,7 @@ compileJava {
 }
 
 javafx {
+    version '11'
```

Comment:
Why 11 and not 15?

Reply:
I wanted to keep it in sync with Java version :)

Reply:
I don't see a reason for it. We should try to use the latest JavaFX release in all our libraries?